llvm/lib/Transforms/Scalar/LoopStrengthReduce.cpp

```diff
@@ -81,6 +81,7 @@
 #include "llvm/Analysis/TargetTransformInfo.h"
 #include "llvm/Analysis/ValueTracking.h"
 #include "llvm/BinaryFormat/Dwarf.h"
+#include "llvm/CodeGen/Passes.h"
```

Comment:
Is this needed? Looks the other changes in this file were deletions so I don't understand why a new header is needed.